spanner/spanner_snippets/spanner/spanner_create_backup.go

```diff
@@ -28,7 +28,7 @@ import (
 	adminpb "google.golang.org/genproto/googleapis/spanner/admin/database/v1"
 )
 
-func createBackup(w io.Writer, db, backupID string) error {
+func createBackup(w io.Writer, db, backupID string, versionTime time.Time) error {
```

Comment:
Will you add sample values for these arguments? See https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#function-arguments-for-snippets.

Reply:
Added a sample value for the `versionTime`.